misc/faucet_dot1x_nodot.yaml

```diff
@@ -0,0 +1,19 @@
+dps:
```

Comment:
nit: other .yaml files seem to be 2-space indent.  Doesn't really matter, but generally good to maintain consistency.  No need to change this now, but just something to be aware of the next time you create a file!